numpy/linalg/_linalg.py

```diff
@@ -2087,17 +2101,21 @@ def pinv(a, rcond=1e-15, hermitian=False):
     ----------
     a : (..., M, N) array_like
         Matrix or stack of matrices to be pseudo-inverted.
-    rcond : (...) array_like of float
+    rcond : (...) array_like of float, optional
         Cutoff for small singular values.
         Singular values less than or equal to
         ``rcond * largest_singular_value`` are set to zero.
-        Broadcasts against the stack of matrices.
+        Broadcasts against the stack of matrices. Default: ``1e-15``.
     hermitian : bool, optional
         If True, `a` is assumed to be Hermitian (symmetric if real-valued),
         enabling a more efficient method for finding singular values.
         Defaults to False.
 
         .. versionadded:: 1.17.0
+    rtol : (...) array_like of float, optional
+        Same as `rcond`, but it's an Array API compatible parameter name.
+        Only `rcond` or `rtol` can be set at a time. If none of them are
+        provided then ``1e-15`` default is used.
```

Comment:
Needs a `.. versionadded:: 2.0.0`

Reply:
Done!

Reply:
Should note here that if `None` is passed in, the API standard default is used.

Reply:
Done!